src/components/pages/donation_confirmation/SuccessMessage.vue

```diff
@@ -43,6 +43,18 @@ export default defineComponent( {
 			} );
 		},
 		donationSummaryMessage(): String {
+			if ( ( this.$props.donation.paymentType === 'MCP' ||
```

Comment:
You can do `if ( this.$props.donation.interval === 0 && [ 'MCP', 'PPL', 'SUB' ].includes( this.$props.donation.paymentType )`

Putting the interval check first makes the code a teeny bit faster because if interval is not 0 then it won't even try to evaluate the paymentType.

And using an `array.includes` check makes it a little more readable.

Reply:
Thank you for this suggestion.